whirl

```diff
@@ -122,9 +150,25 @@ test_dag_state() {
   fi
 }
 
+test_dag_errors() {
+  local DAG_ID=$1
+  local SHOW_RESULT=${2:-false}
+  echo "Checking dag ${DAG_ID} for errors." >&2
+  result=$(curl -s -u admin:admin "http://localhost:${AIRFLOW_UI_PORT}/api/v1/importErrors" | jq ".total_entries")
```

Comment:
would it make sense to move some of this to global variables in this script? Would reduce the amount of change required if we ever need/want to change username/password/api endpoint?

Reply:
Created separate issue for this